src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,6 +1,20 @@
 package frc.robot;
 
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.units.Unit;
+
 public class RobotMap {
+    public static final double WHEEL_RADIUS = Units.inchesToMeters(3);
+
+    public static final int LEFT_FRONT_MOTOR_ID = 4;
```

Comment:
```suggestion
    public static final int DRIVE_LEFT_FRONT_MOTOR_ID = 4;
```
Be specific with names, we have a ton a motors on the robot.
Same for all the Constants below which are specific to drive.